Header header
sensor_msgs/Image rgb
sensor_msgs/Image depth
sensor_msgs/CameraInfo rgb_info
motion_rect[] motion_rects
